Add -lm to libs to avoid undefined symbol _ZGVbN2v_lo
Patch by Simon Branford (University of Birmingham)
--- src/Makevars.orig	2021-06-06 11:55:48.233875000 +0100
+++ src/Makevars	2021-06-06 11:55:58.272198000 +0100
@@ -1 +1 @@
-PKG_LIBS = $(LAPACK_LIBS) $(BLAS_LIBS) $(FLIBS)
+PKG_LIBS = $(LAPACK_LIBS) $(BLAS_LIBS) $(FLIBS) -lm
